app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,10 @@
+class StaticPagesController < ApplicationController
+  def home
+  end
+
+  def help
```

Comment:
```suggestion
  def help; end
```
Tương tự, Cái này k có body chưa viết code xử lý thì viết 1 line như vậy